file_paths.md

```diff
@@ -275,6 +279,12 @@ create a `puppet` user or group.
         lib *
             puppet.rb *
 
+    C:\Program Files\Puppet Labs\Puppet\pxp-agent *
+        bin *
```

Comment:
It's not necessary. Not sure if it makes sense for facter either tbh. Neither `inc` directory is hurting much either so if it's at all difficult to *not* ship it, I'd be happy to defer that activity.